core/storage/classifier/gae_models.py

```diff
@@ -26,6 +26,11 @@
     'LDAStringClassifier'
 ]
 
+# Permitted training job status values
+TRAINING_JOB_STATUSES = [
+    'PENDING',
```

Comment:
You probably want to use named constants (TRAINING_JOB_STATUS_PENDING, etc.) for the individual elements, too; you'll be reusing those in the future.